docs/usage/exporting_models.md

```diff
@@ -64,15 +69,22 @@ print(Model(x=['{"a": 1}', '[1, 2]']).model_dump(round_trip=True))
 #> {'x': ['{"a":1}', '[1,2]']}
 ```
 
-## `dict(model)` and iteration
+## `model.model_dump_json(...)`
 
-*pydantic* models can also be converted to dictionaries using `dict(model)`, and you can also
-iterate over a model's field using `for field_name, value in model:`. With this approach the raw field values are
-returned, so sub-models will not be converted to dictionaries.
+The `.model_dump_json()` method will serialize a model to JSON.
 
-Example:
+The behavior of this method is similar to `.model_dump` (including how it handles
+[`RootModel`](models.md#custom-root-types) and subclasses), just that the returned result will be a JSON-encoded string.
```

Comment:
I thought for RootModel in JSON there is no `__root__`/`root` key?

Reply:
There is no `root` key in `json`. It's the same as for `model_dump`. So, `RootModel.model_dump()` doesn't have the `root` field and neither `RootModel.model_dump_json()` does.

IMO, this documentation doesn't contradict this.

Reply:
How would you guys feel about me replacing lines 74-77 with:

```
The `.model_dump_json()` method serializes a model directly to a JSON-encoded string that is equivalent to the result produced by `.model_dump`.
```

I feel like this addresses the confusion, and is more terse to boot. If I get a 👍 I'll push the change (couldn't do a suggestion since deleted lines are in the span)

Reply:
I made the change since I was working on this to resolve conflicts anyway. Happy to reconsider if appropriate